domains/qrcode.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "My QRCode Webpage Project",
+  "owner": {
+    "username": "isobarsmustang0g",
+    "email": "isobars.mustang_0g@icloud.com"
+  },
+  "record": {
+    "A": [
+      "103.97.126.29"
```

Comment:
I cannot reach your website, keeps on loading (if it's on a different port, can you show a preview?)

Reply:
Here
![image](https://github.com/user-attachments/assets/aed022ea-465e-4e99-ad06-2c249e2b0e4f)

- My website is provided by a Vietnamese company, so you may not be able to reach it because they block all connections except those from Vietnam.